### PR TITLE
#landing-page: users landing page

### PR DESCRIPTION
**Pull request by nraufu**
> #### What does this PR do?
- It creates a landing page for users of the website
> #### Description of Task to be completed
- add folder structure (css,js,index.html)
- add a navbar with links to visit other pages
- add h1 tags containing small information about the developer
> #### How should this be manually tested?
- git clone the project
- open the index.html file in the browser you should see the page
> #### What are the relevant Trello stories?
- https://trello.com/c/EcWmS32m/1-user-landing-page
#### Screenshots (if appropriate)
landing__page

### CODE REVIEW

**Comment by T2Wil** [comment]
Hey @nraufu , 
You did nice work, here are the few feedback you might consider working on:
- on branch name: consider adding story id.
- in PR description: add your git link to make it easy for a reviewer cloning your repo.
- Mobile responsiveness: make the web responsive on small devices.

File: UI/css/style.css [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    overflow: hidden;
+}
+
+.navigation {
+    position: relative;
+    display: flex;
+    justify-content: space-between;
+    background-color: rgba(255, 255, 255, 0.8);
+    box-shadow: 0 .2rem 2rem rgba(0, 0, 0, 0.25);
+    z-index: 500;
```
Comment (T2Wil) [inline UI/css/style.css]: why is z-index having such big value?

Comment (nraufu) [reply UI/css/style.css]: I just putted it like to verflow those three bar image.

Comment (nraufu) [reply UI/css/style.css]: okay let start the mobile responsive part

**Comment by nraufu** [comment]
[User Landing Page](https://trello.com/c/EcWmS32m/1-user-landing-page)

**Pull request merged**